crates/polars-core/src/chunked_array/comparison/mod.rs

```diff
@@ -978,18 +978,22 @@ impl BooleanChunked {
         self.downcast_iter().any(compute::boolean::any)
     }
 
-    // Three-valued versions which can return None
-    pub fn all_3val(&self, drop_nulls: bool) -> Option<bool> {
-        if drop_nulls || self.null_count() == 0 {
+    // Three-valued versions which can return None.
+    pub fn all_3val(&self, ignore_nulls: bool) -> Option<bool> {
+        // self.downcast_iter().map(|v|)all(compute::boolean::all)
```

Comment:
Commented out code

Reply:
Yeah I had some cleanup to do still! (PR was still in draft 😸 )